content/how-to/fabric/fabric-ticketing/index.md

```diff
@@ -0,0 +1,153 @@
+
+<img width="621" alt="FT13" src="https://user-images.githubusercontent.com/98875667/156184075-2b34647c-1c4d-4053-b4ea-20486b52b52f.png">
+
+<img width="374" alt="FT8" src="https://user-images.githubusercontent.com/98875667/156186872-21df8151-9e8f-44f2-abb4-c9fca88e1902.png">
+
+You can load your ticket in a new browser tab by selecting the ticket ID from the header of the right ticketing panel as shown in above image.
+
+The right-side ticket panel displays the details of the selected ticket. The details of the ticket are divided into three tabs such as: 
+
+* **Comments**: Comment option displays the details of the most recently created ticket. 
+
+<img width="691" alt="FT14" src="https://user-images.githubusercontent.com/98875667/156187554-fc47f74f-89a0-44b5-865c-ccbb90353123.png">
+
+* **Details**: You can view the details of the tickets in terms of the category, created by, modified by, severity, created date, recipients and so forth.
+
+<img width="290" alt="FT15" src="https://user-images.githubusercontent.com/98875667/156187836-c9be71b9-a9f2-408f-a918-84cb46e277f6.png">
+
+You can **Add Recipient** to the ticket as shown in the above image. The added recipients will get notification about the ticket once added. The **Add Recipient** pop over will hide contacts that have ticket notifications settings disabled on a dedicated account and does not apply to multi-cloud contacts.
+
+<img width="280" alt="FT16" src="https://user-images.githubusercontent.com/98875667/156188912-77a59f2c-d667-4038-afa3-f4c4747a4db1.png">
+
+You can **Add Groups** to the ticket as shown in the above image. The added groups will get notifications about the ticket. This option is available for the dedicated users.
+
+<img width="308" alt="FT17" src="https://user-images.githubusercontent.com/98875667/156189256-474c010b-a7db-4471-ae60-677c338d03cf.png">
+
+* **Devices**: You can view the device list. This option is only applicable for dedicated customer tickets at this current time. 
+
+## How To Create Ticket
+
+The create ticket is different as compared to the previous MyRack ticketing application that dedicated customers are familiar with. The Fabric Ticketing Application automatically determines which queue or team a ticket will be sent to. This is calculated based on the user’s selection of the category, account type, product/service, and device(s). Most of the selections are within Step: 2.
+
+## Step 1: Choose Category
+
+![Choose Category](https://user-images.githubusercontent.com/98875667/156199904-a4cf30a8-3009-48bc-9847-c631db7059a1.png)
+
+You must choose the required category from the pre-defined ITIL/ITSM management options. Below is the list of categories:
+
+* **Change**: Change category is used to raise the ticket for add, remove or modify the infrastructure. Examples of change category are Configuration of Manager Console, add               or remove hosts, and so on. 
```

Comment:
@asmita32675 do we really need the extra spaces between `add               or` ?
Also double check the other categories below for the same thing.

Reply:
I have removed the extra spaces, found at two instance. Checked entire doc.